java-devel/Dockerfile

```diff
@@ -53,8 +53,6 @@ RUN microdnf update && \
     which \
     xz \
     zlib-devel \
-    # Required for log4shell.sh
```

Comment:
The diff looks like it is now missing a `&&`

Reply:
[`d5ef172` (#1192)](https://github.com/stackabletech/docker-images/pull/1192/commits/d5ef1725100fb682f27a9a18565167608a69070d)